bindings.cpp

```diff
@@ -2458,6 +2471,8 @@ EMSCRIPTEN_BINDINGS(testBinding) {
 
     function("adaptiveThreshold", select_overload<void(const cv::Mat&, cv::Mat&, double, int, int, int, double)>(&Wrappers::adaptiveThreshold_wrapper));
 
+    function("magnitudeThreshold", select_overload<void(const cv::Mat&, const cv::Mat&, cv::Mat&, double, double, double)>(&Wrappers::magnitudeThreshold_wrapper));
```

Comment:
Are above changes specific to wasm support? Otherwise, separate them into another commit?